File: ignored-test/csharp-xunit/ClassOne_TestOneOneIgnored.cs
using Xunit;

namespace TestAssetsXunit
{
    public class ClassOne
    {
        [Fact(Skip = "")]
        public void TestOneOneIgnored()
        {
            {
                Assert.True(1 == 2, "This is ignored, no failure");
            }
        }
    }
}

File: ignored-test/java-junit/ClassOne_TestOneOneIgnored.java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Disabled;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class ClassOne {

    @Test
    @Disabled("")
    public void testOneOneIgnored() {
        assertTrue(1 == 2, "This is ignored, no failure");
    }
}
